modules/mithril-shell.nix

```
@@ -91,6 +91,14 @@ in
             percentage is never shown if no battery is detected.
           '';
         };
+        workspacesIndicator.reverseScrollDirection = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Reverse the scroll direction of the workspaces indicator, for all you natural
+            scrollling users.
+          '';
+        }
```

You seem to have a small syntax error here
```suggestion
        };
```

---

Fix missing semicolon in `mithril-shell.nix`